### ISSUE TITLE
Add configuration to support different (than en.yml) base file. 

### ISSUE THREAD

**Issue by ekampp** [narralabs/i18n-ai#3]
It would be awesome if this gem easily supported selecting a different locale as the base of the translations. E.g. an it.yml file instead of the en.yml one that's hard coded today.

**Comment by westoque** [comment]
that's correct. that is definitely the plan to support a different base file besides en.yml

**Comment by ekampp** [comment]
I'm happy to give it a shot with a PR when I get back to a computer, if you're open to that?
